src/classes/EDAHealthCheckGroupAPIService.cls

```diff
@@ -64,164 +64,29 @@ public virtual with sharing class EDAHealthCheckGroupAPIService implements Healt
-                                                  hlthChkItemsList);
-            }
-
-            when 'all failure' {
-                rowKeysList.add('01');
-
-                HealthCheckItemVModel hlthChkItemMod1_1 = 
-                    new HealthCheckItemVModel('01.1', 
-                                            'Failed Nested Setting 1', 
-                                            'Fail', 
-                                            Label.stgHealthCheckStatusFailed, 
-                                            'Mocked nested description 1', 
-                                            'Mocked nested recommended fix 1',
-                                            new List<HealthCheckItemVModel>());
-
-                HealthCheckItemVModel hlthChkItemMod1_2 = 
-                    new HealthCheckItemVModel('01.2', 
-                                            'Failed Nested Setting 2', 
-                                            'Fail', 
-                                            Label.stgHealthCheckStatusFailed, 
-                                            'Mocked nested description 2', 
-                                            'Mocked nested recommended fix 2',
-                                            new List<HealthCheckItemVModel>());
-
-                HealthCheckItemVModel hlthChkItemMod1 = 
-                    new HealthCheckItemVModel('01', 
-                                              'Failed Setting 1', 
-                                              'Fail', 
-                                              Label.stgHealthCheckStatusFailed, 
-                                              'Mocked description 1', 
-                                              'Mocked recommended fix 1',
-                                              new List<HealthCheckItemVModel>{hlthChkItemMod1_1, hlthChkItemMod1_2});
-
-                HealthCheckItemVModel hlthChkItemMod2 = 
-                    new HealthCheckItemVModel('02',
-                                              'Failed Setting 2',
-                                              'Fail',
-                                              Label.stgHealthCheckStatusFailed,
-                                              'Mocked description 2',
-                                              'Mocked recommended fix 2',
-                                              new List<HealthCheckItemVModel>());
-
-                hlthChkItemsList.add(hlthChkItemMod1);
-                hlthChkItemsList.add(hlthChkItemMod2);
-
-                return new HealthCheckGroupVModel('All Failure',
-                                                  'Fail',
-                                                  0,
-                                                  3,
-                                                  rowKeysList,
-                                                  hlthChkItemsList);
+            when 'reciprocalrelationship' {
```

Comment:
This works but maybe a final upper case variable value would be a better option.

Reply:
Oh wait, wrong PR. More coffee for me.